migration-roadmap/checkbox.md

```diff
@@ -0,0 +1,284 @@
+| `.spectrum-Checkbox--sizeS`                                                                 | `size="s"` attribute                                 | Implemented      |
+| `.spectrum-Checkbox--sizeL`                                                                 | `size="l"` attribute                                 | Implemented      |
+| `.spectrum-Checkbox--sizeXL`                                                                | `size="xl"` attribute                                | Implemented      |
+| `.spectrum-Checkbox.is-indeterminate`                                                       | `indeterminate` attribute                            | Implemented      |
+| `.spectrum-Checkbox.is-invalid`                                                             | `invalid` attribute                                  | Implemented      |
+| `.spectrum-Checkbox-input`                                                                  | Internal checkbox input element, `#input`            | Implemented      |
+| `.spectrum-Checkbox-input:checked`                                                          | `checked` attribute                                  | Implemented      |
+| `.spectrum-Checkbox-input:disabled`                                                         | `disabled` attribute                                 | Implemented      |
+| `.spectrum-Checkbox-box`                                                                    | Internal checkbox box element, `#box`                | Implemented      |
+| `.spectrum-Checkbox-checkmark`                                                              | Checkmark icon element, `#checkmark`                 | Implemented      |
+| `.spectrum-Checkbox-partialCheckmark`                                                       | Indeterminate dash icon element, `#partialCheckmark` | Implemented      |
+| `.spectrum-Checkbox-label`                                                                  | Default slot content                                 | Implemented      |
+| `.spectrum-Checkbox:lang(ja)`, `.spectrum-Checkbox:lang(ko)`, `.spectrum-Checkbox:lang(zh)` | Language-specific styling                            | Implemented      |
+| `.spectrum-Checkbox.is-readOnly`                                                            | `readonly` attribute                                 | Implemented      |
+|                                                                                             | `name` attribute                                     | Missing from CSS |
+|                                                                                             | `tabindex` attribute                                 | Missing from CSS |
+|                                                                                             | `autofocus` attribute                                | Missing from CSS |
```

Comment:
Are these something we think is worth mentioning? Would CSS ever come back and implement these attributes?